src/components/MetaData/index.tsx

```diff
@@ -439,18 +513,17 @@ const MetaData: React.FC<Props> = ({
     </>
   );
 
-  // Canonical Link
-  const canonicalDomainMapping = {
-    article: canonicalDomain,
-    video: canonicalDomain,
-    gallery: canonicalDomain,
-  };
+  if (outputCanonicalLink) {
+    const defaultResolution = getPageCanonicalUrl(pageType, canonicalDomain || websiteDomain, gc, requestUri);
```

Comment:
@nschubach To my previous point around `canonicalDomain || websiteDomain` - above there is a comment against the `canonicalDomain` that is ```/** The canonical domain name to be used for article, video and gallery pages */```

The way I am reading this updated implementation is there is no need to distinguish between a `canonicalDomain` and `websiteDomain` within this code as there is no logic that uses one over the other. The calling code would have to ensure for the page that's being rendered to ensure the props are set correctly. So we end up with in the output type having code like this

```
<MetaData
   canonicalDomain={pageType == 'article || pageType == 'video' || pageType == 'gallery' ? http://canonical-domain.com : null}
   websiteDomain={pageType != 'article || pageType != 'video' || pageType != 'gallery' ? http://website-domain.com : null}
```

Reply:
Would you be ok with me changing the comment to say: "An override for websiteDomain when rendering canonical metadata links"?  I know this was added recently.  Is there a client using that field for something specific to article, video, and gallery?

Reply:
It's not for any given client - but for all that have stories that are shown on a site that has a different canonical URL.
An example is if you view this story - http://localhost/local/2020/01/14/tampa-reinvents-itself-to-show-off-its-shores/?_website=the-sun
the canonical url tag should be using the-gazette's domain as the website domain. Which is fine - but that logic will need to be pushed up to the output type if not handled here

Reply:
Change is present in the related PR https://github.com/WPMedia/fusion-news-theme-blocks/pull/1174